modules/virt-about-aaq-operator.adoc

```diff
@@ -0,0 +1,77 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/advanced_vm_management/virt-understanding-aaq-operator.adoc              
+
+:_mod-docs-content-type: CONCEPT                                    
+[id="virt-about-aaq-operator_{context}"]                                   
+= About the AAQ Operator
+
+
+The AAQ Operator provides more flexible and extensible quota management compared to the native `ResourceQuota` object in the {product-title} platform.
```

Comment:
Should AAQ be expanded on every first-use instance in a module?

Reply:
I don't see the need to spell it out in every module. Thanks for the suggestion.